test/mail/renderers/rfc_2822_test.exs

```diff
@@ -42,6 +42,11 @@ defmodule Mail.Renderers.RFC2822Test do
     assert headers == "Foo: bar\r\nBaz: qux"
   end
 
+  test "headers - handles empty headers" do
+    headers = Mail.Renderers.RFC2822.render_headers(%{"content-type" => "text/plain", "message-id" => nil, "to" => [], "from" => "      ", "content-disposition" => "attachment"})
```

Comment:
Hmm, I did handle all three cases in that line, but I guess splitting them would be the correct approach